spock_apply.c

```diff
@@ -1843,6 +1870,7 @@ handle_startup(StringInfo s)
 static void
 spock_apply_worker_on_exit(int code, Datum arg)
 {
+	/* FIXME: Do we really need to flush the sync_replica_lsn*/
```

Comment:
We should perform the cleanup on shared memory exit to ensure that:
(1) we are sending back the feedback confirmation to the publisher,
(2) cleaning up the list.